src/main/java/pl/tomaszdziurko/jvm_bloggers/blogs/BloggersDataFetcher.java

```diff
@@ -23,6 +23,7 @@
     private final Optional<URL> bloggersUrlOptional;
     private final Optional<URL> companiesUrlOptional;
     private final BloggersDataUpdater bloggersDataUpdater;
+    private final ObjectMapper mapper = new ObjectMapper();
```

Comment:
Why did you move the `mapper` to global scope if it's only used in the `refreshBloggersDataFor`(https://github.com/tdziurko/jvm-bloggers/pull/176/files#diff-492313bd8196a0a1a2ae2b2e32a15e5cL55) scope?